Marble Blast Platinum/platinum/server/scripts/game.cs

```diff
@@ -1437,6 +1437,37 @@
 	return %nearest;
 }
 
+
+function GameConnection::getHighestValueNearestGem(%this) {
+	if (isObject(%this.player)) {
+		return getHighestValueNearestGem(%this.player.getTransform());
+	}
+	return getHighestValueNearestGem("0 0 0");
+}
+
+// main_gi: New function for respawns to point at.
```

Comment:
i'd probably remove the 2 comments here, as the code explains it well enough what its doing. comments usually are meaningless and clutter code.

Reply:
I agree that the second comment in the `%highest` loop is unnecessary, but I'm going to defend the use of this comment in game.cs, because I don't think it's necessarily clear that the function of "highest value gem" in game.cs is associated with respawning.